openskistats/plot.py

```diff
@@ -176,15 +185,19 @@ def plot_orientation(
         _mpl_add_polar_margin_text(ax=ax, ylim=ylim, location=location, text=text)
 
     if title:
+        font_path: Path = get_font_path(TITLE_FONT, keywords=[TITLE_WEIGHT])
+        if font_path:
+            print(f"Font path: {font_path}")
+        else:
+            print(f"Font not found at {font_path}")
+        prop = font_manager.FontProperties(fname=font_path)
+
         ax.set_title(
             title,
             y=title_y,
-            fontdict={
-                # fallback font families for more character support https://matplotlib.org/stable/users/explain/text/fonts.html
-                "family": ["DejaVu Sans", "Noto Sans CJK JP"],
```

Comment:
Copying from the readme PR, I think when Noto Sans CJK JP is not found on the system, it's a warning not an error:

```
WARNING:matplotlib.font_manager:findfont: Font family 'Noto Sans CJK JP' not found.
```

The intention prior to this PR is to use DejaVu Sans as the primary font and then to fallback to Noto Sans CJK JP for unsupported characters, mostly chinese, japenese, korean, etc.

It's not a big problem IMO if users are missing Noto Sans CJK JP locally as the website gets deployed on GitHub actions where we do install this font:

https://github.com/dhimmel/openskistats/blob/9aea7f63dded36cdab39bae9b17d19d3a7a7cde2/.github/workflows/analyze.yaml#L23

Help me understand what this PR is trying to do.

Reply:
The warnings make uv run openskistats analyze unreadable relative the the outputs that are actually informative here.

Reply:
And even when I installed the font fonts-noto-cjk I couldn't suppress the error until I directly set the font path instead of using the fallback list   "family": ["DejaVu Sans", "Noto Sans CJK JP"],

Reply:
So prior to this PR, when you run:

```
from matplotlib import font_manager
font_manager.get_font_names()
```

what do you see?